src/network/clientpackethandler.cpp

```diff
@@ -1102,22 +1102,17 @@ void Client::handleCommand_HudAdd(NetworkPacket* pkt)
 	v3f world_pos;
 	v2s32 size;
 	s16 z_index = 0;
+	std::string text2;
 
 	*pkt >> server_id >> type >> pos >> name >> scale >> text >> number >> item
 		>> dir >> align >> offset;
 	try {
 		*pkt >> world_pos;
-	}
-	catch(SerializationError &e) {};
-
-	try {
 		*pkt >> size;
-	} catch(SerializationError &e) {};
-
-	try {
 		*pkt >> z_index;
+		*pkt >> text2;
 	}
-	catch(PacketError &e) {}
+	catch(SerializationError &e) {};
```

Comment:
Needs to catch `PacketError`. Why was this changed?
Discovered while testing another PR.